.github/dependabot.yaml

```diff
@@ -5,3 +5,8 @@ updates:
     schedule:
       interval: 'weekly'
     open-pull-requests-limit: 5
+    groups:
+      production-dependencies:
+        dependency-type: 'production'
+      development-dependencies:
+        dependency-type: 'development'
```

Comment:
group pull request, 
- one for dependencies,
- another one for dev-dependencies